src/OceanSeaIceModels/InterfaceComputations/assemble_net_fluxes.jl

```diff
@@ -239,7 +243,7 @@ end
     Qu = upwelling_radiation(Ts, σ, ϵ)
     Qd = net_downwelling_radiation(i, j, grid, time, α, ϵ, Qs, Qℓ)
 
-    ΣQt = Qd + Qu + Qc + Qv
+    ΣQt = (Qd + Qu + Qc + Qv) * ℵi # If ℵi == 0 there is no heat flux from the top!
```

Comment:
actually I am not completely sure if this must be multiplied by `ℵi` or it should be
```julia
ΣQt = ifelse(ℵi == 0, zero(grid), Qd + Qu + Qc + Qv)
```
for sure if there is no sea ice `ΣQt` should be zero, but if there is, probably `ℵi` does not play a role because the sea ice thickness is computed per column and not per area.

Reply:
is this the net flux into sea ice?

Reply:
yeah, from the atmosphere

Reply:
yes, in or out of the sea ice from the atmosphere

Reply:
before it was becoming negative at night and because sea-ice was T=0C it was forming sea ice in the equator!

but I also agree it should be ifelse, not multiply by ℵi

Reply:
I think this is the same problem https://github.com/CliMA/ClimaSeaIce.jl/pull/57
Here I was multiplying times the concentration. I will try the `ifelse` in this PR

Reply:
or

```julia
(Qd + Qu + Qc + Qv) * (ℵi == 0)
```

which I think is easier to read.